runbotics/runbotics-desktop/src/action/desktop/desktop.action-handler.ts

```diff
@@ -0,0 +1,124 @@
+import { StatelessActionHandler } from 'runbotics-sdk';
+import { RunboticsLogger } from '#logger';
+import {
+    mouse,
+    keyboard,
+    straightTo,
+    Point,
+    Button,
+    Key,
+} from '@nut-tree/nut-js';
+import * as DesktopTypes from './types';
+
+export default class DesktopActionHandler extends StatelessActionHandler {
+    private readonly logger = new RunboticsLogger(DesktopActionHandler.name);
+
+    constructor() {
+        super();
+    }
+
+    private checkIfNaN(xCoordinate: number, yCoordinate: number) {
+        if (isNaN(xCoordinate) || isNaN(yCoordinate)) {
+            throw new Error('Element coordinates have to be of type number');
+        }
+    }
+
+    async click(input: DesktopTypes.DesktopClickActionInput) {
```

Comment:
it's similar to excel action handler

here are only desktop actions, so imo import types directly - remove DesktopTypes.xxxx